### PR TITLE
Better SEO

### PR DESCRIPTION
**Pull request by jasonlemay**
- Meaningful titles and descriptions for every page
- Add social share images

### CODE REVIEW

File: site/src/main/markdown/config.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE folder SYSTEM "folder.dtd">
+<folder displayName="Arcbees platform for developers"
+        description="GWTP, Jukito, GQuery, GWT-Chosen, GSSS, Universal Analytics, GAE Studio."
+        logo="assets/build/img/facebook.png">
+    <toc>
+        <entries>
+            <entry name="index" displayName="Arcbees platform for developers" title="Arcbees platform for developers"
```
Comment (olafleur) [inline site/src/main/markdown/config.xml]: The Arcbees platform? Arcbees' platform?

File: site/src/main/markdown/gaestudio/index.md [hunk 2]
```diff
@@ -25,30 +25,26 @@ GAE Studio will be available at http://gaestudio.__application_id__.appspot.com
 * Create a file named `application.xml` in `/src/META-INF`
 
     ```xml
-        <?xml version="1.0" encoding="UTF-8"?>
+        <?xml version="1.0" encoding="utf-8"?>
```
Comment (olafleur) [inline site/src/main/markdown/gaestudio/index.md]: I'm curious. What does that change?

File: site/src/main/markdown/gaestudio/index.md [hunk 3]
```diff
@@ -154,9 +143,3 @@ GAE Studio is already declared as a module called 'gaestudio'.
 You can run this EAR using the Java Appengine Tools (Google Appengine SDK for Java): /appengine-java-sdk-x.y.z/bin/dev_appserver.sh /path/to/my/ear.
 The development server will start your app and GAE Studio on two different ports. They will be displayed in the
 console output. ie: INFO: Module instance gaestudio is running at http://localhost:8081/
-
-##Running GAE Studio locally (Python runtime)
```
Comment (olafleur) [inline site/src/main/markdown/gaestudio/index.md]: Yeah. For the moment it should be removed I agree. But we should ideally write these sections some day. ;-)

File: site/src/main/markdown/gwtp/contributing/Compiling-and-Debugging.md [hunk 4]
```diff
@@ -86,11 +86,11 @@ If you want to build the javadoc yourself, all you have to do is to activate the
 
 The javadoc will be under `gwtp-core/*/target/apidocs`, simply open `index.html` in there. You can also find the javadoc for the current SNAPSHOT version of gwt-platform on the [ContinuousIntegration Continuous Integration server].
 
-##Making sure the Eclipse checkstyle plugin is running
+### Making sure the Eclipse checkstyle plugin is running
 
 *TODO: Explain how this should work for Eclipse 3.7*
```
Comment (olafleur) [inline site/src/main/markdown/gwtp/contributing/Compiling-and-Debugging.md]: I think you can remove that comment.

**Comment by olafleur** [comment]
LGTM

File: site/src/main/markdown/gwtp/advanced/Merge-Locale.md [hunk 5]
```diff
@@ -18,16 +18,8 @@ The exact process is as follows:
...
-the directory of any super interfaces. The problem with `UiBinder`
-translations is that the associated `LocalizableResource` files are
-generated in the folder of the `UiBinder` file, so your resources are
-spread across your directories. `mergelocales` looks for all of these and
-merges them intelligently in a central location: com.google.gwt.i18n.client. This is legit since this package is the
-directory of the `LocalizableResource`, a superinterface common to both
-Constants and Messages.
+## Additional information
+GWT looks for the `LocalizableResource???.properties` files in the directory of your class implementing Constants or Messages, and in the directory of any super interfaces. The problem with `UiBinder`translations is that the associated `LocalizableResource` files aregenerated in the folder of the `UiBinder` file, so your resources arespread across your directories. `mergelocales` looks for all of these andmerges them intelligently in a central location: com.google.gwt.i18n.client. This is legit since this package is thedirectory of the `LocalizableResource`, a superinterface common to bothConstants and Messages.
```
Comment (meriouma) [inline site/src/main/markdown/gwtp/advanced/Merge-Locale.md]: space between `Uibinder` and `translations`

Comment (meriouma) [inline site/src/main/markdown/gwtp/advanced/Merge-Locale.md]: space between `are` `spread`

Comment (meriouma) [inline site/src/main/markdown/gwtp/advanced/Merge-Locale.md]: space between `and` `merges`

Comment (meriouma) [inline site/src/main/markdown/gwtp/advanced/Merge-Locale.md]: space between `are` `generated `

Comment (meriouma) [inline site/src/main/markdown/gwtp/advanced/Merge-Locale.md]: space between `the` `directory`

Comment (meriouma) [inline site/src/main/markdown/gwtp/advanced/Merge-Locale.md]: space between `both` `Constants`

File: site/src/main/markdown/gwtp/advanced/config.xml [hunk 6]
```diff
@@ -3,12 +3,18 @@
...
+            <entry name="Phonegap-support" title="Phonegap Support" displayName="Phonegap Support"
+                    description="How to use PhoneGapDispatchAsync with DefaultDispatchAsync for deploying an application to PhoneGap."/>
+            <entry name="crawler-support" title="Crawler Support" displayName="Crawler Support"
+                    description="GWTP makes it easy for your website to be crawlable by search engines that follows Google's AJAX crawling specification."/>
+            <entry name="Custom-Proxy-Class" title="Custom Proxy Class" displayName="Custom Proxy Class"
+                    description="In rare cases, it might be useful to code your own custom Proxy class."/>
+            <entry name="Google-Universal-Analytics" title="Google Universal Analytics" displayName="Google Universal Analytics"
+                    description="The Universal Analytics library lets you make calls to Google Analytics directly from Java code."/>
+            <entry name="Merge-Locale" title="Merge Locale" displayName="Merge Locale"
+                    description="GWTP makes it easy for you to store all your translations with UIBinder internationalisation features."/>
```
Comment (meriouma) [inline site/src/main/markdown/gwtp/advanced/config.xml]: internationalization I think

File: site/src/main/markdown/gwtp/communication/action-validators.md [hunk 7]
```diff
@@ -79,7 +79,7 @@ The only change here is that we don't use hard coded dispatch string in your mod
 serve("/yourappname/" + ActionImpl.DEFAULT_SERVICE_NAME + "*").with(DispatchServiceImpl.class);
 ```
 
-#!ActionValidators
+## !ActionValidators
```
Comment (meriouma) [inline site/src/main/markdown/gwtp/communication/action-validators.md]: I believe you can remove the `!`

**Comment by meriouma** [comment]
LGTM

File: site/src/main/markdown/gaestudio/index.md (hunk 2 above)
Comment (jasonlemay) [inline site/src/main/markdown/gaestudio/index.md]: Nothing :)

Comment (olafleur) [inline site/src/main/markdown/gaestudio/index.md]: So why did you make it, then? ;-)

**Pull request merged**